lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -468,6 +516,23 @@ class RenderPopover extends RenderShiftedBox {
   double _constrainFocalPoint(double desiredFocalPoint, ArrowDirection arrowDirection) {
     return min(max(desiredFocalPoint, _minArrowFocalPoint(arrowDirection)), _maxArrowFocalPoint(arrowDirection));
   }
+
+  /// Returns the size which must be reserved to display the arrow.
+  ///
+  /// When [useArrowArea] is `true`, the [child] can take the space that is reserved for the arrow.
+  /// During paint, the [child] is clipped using the border shape.
+  ///
+  /// When [allowHorizontalArrow] is `false`, the reserved width is always zero.
+  Size get _reservedSizeForArrow => Size(
+        (allowHorizontalArrow ? arrowLength * 2 : 0),
+        (useArrowArea ? 0.0 : arrowLength * 2),
+      );
+
+  /// Returns the offset where the border is painted.
```

Comment:
This comment doesn't really help convey what this property represents.

There are a number of ambiguities here:

- You say "painted" but is that really what you mean? Is this property not relevant for layout or hit testing?
- You mention "border" but we're not displaying a border. We're displaying a toolbar, which is comprised of a filled shape and child content. I assume that this offset has something to do with the background shape, but I'm not sure.
- Where does this offset sit in relation to the thing it's referencing? Is this the top left corner?